src/client/assets/styles/components/_command.scss

```diff
@@ -31,6 +31,10 @@ command {
             }
         }
     }
+
+    &.view-mode {
+        pointer-events: none;
```

Comment:
This is for the title bar at the top, which is not part of the monaco-editor. Normally clicking on it would show a menu or something; I've disabled it so there are no effects, hover or clickwise, that show when the user interacts with the div.